src/itinerary/story.tsx

```diff
@@ -25,6 +25,33 @@ const places = [
     stepAriaLabel: 'Dropoff location',
     mainLabel: 'Gare Bordeaux Saint-Jean',
     subLabel: 'Bordeaux',
+    href: '#',
+    connectionLabel: 'Change of vehicle',
+  },
+]
+
+const places2 = [
+  {
+    time: '15:00',
+    isoDate: '2017-12-11T15:00',
+    stepAriaLabel: 'Dropoff location',
+    mainLabel: 'Gare Bordeaux Saint-Jean',
+    subLabel: 'Bordeaux',
+  },
+  {
+    time: '16:00',
+    isoDate: '2017-12-11T12:00',
+    stepAriaLabel: 'Stopover',
+    mainLabel: "Gare d'Agen",
+    subLabel: 'Agen',
+  },
+  {
+    time: '19:00',
+    isoDate: '2017-12-11T19:00',
+    stepAriaLabel: 'Dropoff location',
+    mainLabel: 'Gare De Toulouse',
+    subLabel: 'Toulouse',
+    href: '#',
```

Comment:
I also suggest to add the `actionAriaLabel` on every place as link, like you do on the first place (Paris).
This way the demo will be more coherent.
example here: `actionAriaLabel: '19:00 Gare De Toulouse (New page with a map)`